3rdparty/ymfm/src/ymfm.h

```diff
@@ -325,6 +325,86 @@ struct ymfm_output
 };
 
 
+// ======================> ymfm_wavfile
+
+// this class is a debugging helper that accumulates data and writes it to wav files
+template<int _Channels>
```

Comment:
Please don’t use reserved names – they’re liable to cause issues.  Any identifier that begins with an underscore followed by an uppercase letter is always reserved for the standard library.

For example see here: https://en.cppreference.com/w/cpp/keyword
> Also, all [identifiers](https://en.cppreference.com/w/cpp/language/identifiers) that contain a double underscore __ in any position and each identifier that begins with an underscore followed by an uppercase letter is always reserved and all [identifiers](https://en.cppreference.com/w/cpp/language/identifiers) that begin with an underscore are reserved for use as names in the global namespace. See identifiers for more details.